benchmarks/shelley3pools/configuration/configuration-node-1.yaml

```diff
@@ -114,19 +114,19 @@ traceForwardTo:
 # be specified in the defaults below or overidden on a per-scribe basis here.
 setupScribes:
   - scKind: FileSK
-    scName: "logs/node1.json"
+    scName: "logs/node-1.json"
```

Comment:
please, don't change these names. The analyse scripts and the Readme access the logs via these defined names.

Reply:
Done.